docx/sdt.py

```diff
@@ -22,6 +22,11 @@ def content(self):
     def properties(self):
         return SdtPr(self._element.sdtPr, self)
 
+    @property
+    def is_empty(self):
+        return any((self.properties.active_placeholder,
+                    not self.content.text))
```

Comment:
Does this mean that we want to treat cc empty even if there is some text when `active_placeholder` is set? Doesn't sound quite right to me.